README.md

```diff
@@ -119,6 +119,7 @@ To process raw data, follow these steps:
 
 2. **Set up a Ray cluster**:
     The data processing script relies on Ray for distributed processing of data. This cluster can be either launched on a single node (for small scale data processing) or using AWS EC2 instances.
+    There is also work to [deploy Ray on slurm setups](https://docs.ray.io/en/latest/cluster/vms/user-guides/community/slurm.html), though this effort is still a work-in-progres.
```

Comment:
I feel like this isn't really WIP anymore, in the sense that it is possible to do so if you follow the guide.

Reply:
has one of us been able to get our scripts to work on slurm?

Reply:
I've been launching Ray jobs on TACC, so that part is stable - but I suppose you're right, best not to claim something that might be false